src/tanstack-start.ts

```
@@ -0,0 +1,326 @@
+          : {};
+
+        // Deep merge: override > generated > fallback
+        return this.mergeMetadata(opts.fallback, tanstackHead, opts.override);
+      } catch (error) {
+        console.warn("Failed to get head metadata:", error);
+        return opts.fallback || {};
+      }
+    };
+  }
+
+  public getRootHead<Ctx = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)